components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -461,40 +461,29 @@ interface RunningPrebuildViewProps {
 }
 
 function RunningPrebuildView(props: RunningPrebuildViewProps) {
-    const logsEmitter = new EventEmitter();
-    let pollTimeout: NodeJS.Timeout | undefined;
-    let prebuildDoneTriggered: boolean = false;
+    const [logsEmitter] = useState(new EventEmitter());
 
     useEffect(() => {
-        const checkIsPrebuildDone = async (): Promise<boolean> => {
-            if (prebuildDoneTriggered) {
-                console.debug("prebuild done already triggered, doing nothing");
-                return true;
-            }
-
-            const done = await getGitpodService().server.isPrebuildDone(props.runningPrebuild.prebuildID);
-            if (done) {
-                // note: this treats "done" as "available" which is not equivalent.
-                // This works because the backend ignores prebuilds which are not "available", and happily starts a workspace as if there was no prebuild at all.
-                prebuildDoneTriggered = true;
-                props.onPrebuildSucceeded();
-                return true;
-            }
-            return false;
-        };
-        const pollIsPrebuildDone = async () => {
-            clearTimeout(pollTimeout!);
-            await checkIsPrebuildDone();
-            pollTimeout = setTimeout(pollIsPrebuildDone, 10000);
-        };
-
         const disposables = watchHeadlessLogs(
             props.runningPrebuild.instanceID,
             (chunk) => logsEmitter.emit("logs", chunk),
-            checkIsPrebuildDone,
```

Comment:
this was the only option to proceed, i.e. only if headless logs poller checked for `isPrebuildDone` periodically, we'd a chance to proceed. whenever that poller broke, this was a UX dead-end.